ticket/migrations/0002_auto_20220915_1100.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 2.2.6 on 2022-09-15 11:00
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ticket', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='order',
+            name='fulfilled',
+        ),
+        migrations.AddField(
+            model_name='order',
+            name='state',
+            field=models.CharField(choices=[('CREATED', 'Created'), ('CANCELLED', 'Cancelled'), ('FULFILLED', 'Fulfilled')], default='CREATED', max_length=20),
+        ),
+        migrations.AddField(
+            model_name='order',
+            name='state_change_at',
+            field=models.DateTimeField(null=True),
+        ),
+        migrations.AlterField(
+            model_name='tickettype',
+            name='quantity',
+            field=models.PositiveIntegerField(default=1, editable=False, help_text='The number of actual tickets available upon creation'),
+        ),
+    ]
```

Comment:
This is pure exercise migration. In reality the fields will be first added and then the old values will be migrated and then there will be a series of migrations to remove the filed without any downtime.